src/promptflow/tests/executor/unittests/_core/test_api_injector.py

```diff
@@ -266,7 +266,7 @@ def test_get_aoai_telemetry_headers():
             assert "_" not in key
 
         # assert that the headers are correct
-        assert headers[USER_AGENT_HEADER] == f"test-user-agent promptflow/{VERSION}"
+        assert headers[USER_AGENT_HEADER] == f"promptflow/{VERSION} test-user-agent"
```

Comment:
Please keep test case logic. Not modify to adjust to code change.

Reply:
done